### ISSUE TITLE
Zoom settings don't persist for secondary view

### ISSUE THREAD

**Issue by deepak-rawat** [notepadqq/notepadqq#838]
**Describe the bug**
When notepadqq session with tabs opened in both left and right view close and reopen. The right view start with default zoom level (1.0)

**To Reproduce**
1. Open notepadqq and open few files, increase the zoom level to more than 1.
2. Move some of the files to another view i.e. right view
3. Zoom setting is not applicable to right view
4. Close and reopen notepadqq, saved zoom setting is only applicable to left view

**Expected behavior**
Zoom settings must be consistent across all editor widget

**Screenshots**
https://imgur.com/a/SWJTvTb

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04
 - Installation : git master
 - Qt version : 5.9.5
 - Notepadqq version : 1.4.8

**Additional context**
So I debugged a bit and it seems during load when the zoom setting is restored the second in top editor container is not yet instantiated.

In config user interface
```
    // Restore zoom
    const qreal zoom = m_settings.General.getZoom();
    for (int i = 0; i < m_topEditorContainer->count(); i++) {
        m_topEditorContainer->tabWidget(i)->setZoomFactor(zoom);
    }
```

Even though the notepadqq session has 2 views, here only one count is returned.

**Comment by nikolayhg** [comment]
I can reproduce the issue.

Also, probably related issue:
1. have two views with files
2. Apply higher zoom on the left view
3. Apply lower zoom on the right view
4. Restart notepadqq
5. Now the left view has lower zoom. The right view has the standard zoom

**Comment by deepak-rawat** [comment]
With your step 2 and 3 now it made me think, should session store different zoom factor ? I mean if can have different zoom in/out factor for left and right view, should those be honored when reopen notepadqq.

**Comment by JuBan1** [comment]
At the moment we restore all tabs to the most recently used zoom level (if you ignore this bug in question). Restoring each tab's individual zoom level sounds like a good idea though.

**Comment by nikolayhg** [comment]
yes, if this bug is fixed, then it should still be fine. The user will see the same before and after restart. The atom editor acts like that.
